external-crates/move/move-compiler/tests/move_check/typing/return_any_type.exp

```diff
@@ -6,6 +6,18 @@ warning[W09005]: dead or unreachable code
   │
   = This warning can be suppressed with '#[allow(dead_code)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
 
+warning[W09004]: unnecessary trailing semicolon
+  ┌─ tests/move_check/typing/return_any_type.move:5:24
+  │
+5 │         0 + (return ());
```

Comment:
This feels like a regression. If you remove this `;` the program will fail to typecheck